deltachat-ios/AppDelegate.swift

```diff
@@ -516,15 +537,26 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
 
     // this method will be called if the user tapped on a notification
     func userNotificationCenter(_: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
-        if !response.notification.request.identifier.containsExact(subSequence: Constants.notificationIdentifier).isEmpty {
-            logger.info("Notifications: notification tapped")
-            let userInfo = response.notification.request.content.userInfo
-             if let chatId = userInfo["chat_id"] as? Int,
-                 let msgId = userInfo["message_id"] as? Int {
-                 if !appCoordinator.isShowingChat(chatId: chatId) {
-                     appCoordinator.showChat(chatId: chatId, msgId: msgId, animated: false, clearViewControllerStack: true)
-                 }
-             }
+        logger.info("Notifications: notification tapped")
```

Comment:
What is this log-statement good for?

Reply:
unclear, it was added at some point, maybe for debugging :) i remove it, usually we do not have UI buttons etc. in log